### PR TITLE
Issue#792 delivery stakeholders

### PR DESCRIPTION
**Pull request by Hugo-Hontans**
New design delivery stakeholders

### CODE REVIEW

File: libs/material/src/lib/delivery/components/delivery-stakeholders-repertory/delivery-stakeholders-repertory.component.html [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  <header><h3>Organizations</h3></header>
+  <organization-search (picked)="added.emit($event)"></organization-search>
+  <table mat-table [dataSource]="dataSource" class="mat-elevation-z0">
+    <!-- Name Column -->
+    <ng-container matColumnDef="name">
+      <th mat-header-cell *matHeaderCellDef>Name</th>
+      <td fxLayout="row" fxLayoutAlign="start center" [class.disabled]="!stakeholder.isAccepted" mat-cell *matCellDef="let stakeholder">
+        <!-- TODO: issue#795 add organization logo -->
+        <img
+          src="https://material.angular.io/assets/img/examples/shiba1.jpg"
```
Comment (ioaNikas) [inline libs/material/src/lib/delivery/components/delivery-stakeholders-repertory/delivery-stakeholders-repertory.component.html]: Victor added logo property in his last merge into master. You can use `stakeholder.organization.logo` to display either the organization logo or a placeholder is there is no logo set yet.

File: libs/material/src/lib/delivery/components/delivery-stakeholders-repertory/delivery-stakeholders-repertory.component.html [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+          alt="Stakeholder Picture"
+        />
+        <span>{{ stakeholder.organization.name }}</span>
+      </td>
+    </ng-container>
+
+    <!-- Actions Column -->
+    <ng-container matColumnDef="action">
+      <th mat-header-cell *matHeaderCellDef></th>
+      <td [class.disabled]="!stakeholder.isAccepted" mat-cell *matCellDef="let stakeholder">
```
Comment (ioaNikas) [inline libs/material/src/lib/delivery/components/delivery-stakeholders-repertory/delivery-stakeholders-repertory.component.html]: Not sure we can keep this with dark-theme incoming. We should dismiss it until we got a better solution for pending stakeholders.

File: libs/material/src/lib/delivery/components/delivery-stakeholders-repertory/delivery-stakeholders-repertory.component.scss [hunk 3]
```diff
@@ -0,0 +1,15 @@
+table {
+  width: 100%;
+}
+
+.disabled {
+  background-color: var(--backgroundDisabled);
+  color: var(--textDisabled);
+}
```
Comment (ioaNikas) [inline libs/material/src/lib/delivery/components/delivery-stakeholders-repertory/delivery-stakeholders-repertory.component.scss]: Same as above, we should avoid global css variables to not mess with dark-theme. Maybe just dismiss it until we got a solution.

File: libs/material/src/lib/delivery/pages/delivery-stakeholders-editable/delivery-stakeholders-editable.component.ts [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+@Component({
+  selector: 'delivery-stakeholders-editable',
+  templateUrl: './delivery-stakeholders-editable.component.html',
+  styleUrls: ['./delivery-stakeholders-editable.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class DeliveryStakeholdersEditableComponent implements OnInit {
+  public opened = false;
+
+  public membersFormList = createOrganizationFormList();
```
Comment (ioaNikas) [inline libs/material/src/lib/delivery/pages/delivery-stakeholders-editable/delivery-stakeholders-editable.component.ts]: Do you use it somewhere ?
Plus, name doesn't match content.

File: libs/material/src/lib/delivery/pages/delivery-stakeholders-editable/delivery-stakeholders-editable.component.ts [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+
+  public delivery$: Observable<Delivery>;
+
+  constructor(
+    private deliveryQuery: DeliveryQuery,
+    private service: DeliveryService,
+    private stakeholderService: StakeholderService,
+    private query: DeliveryQuery
```
Comment (ioaNikas) [inline libs/material/src/lib/delivery/pages/delivery-stakeholders-editable/delivery-stakeholders-editable.component.ts]: There is two imports of deliveryQuery. Keep only one.

File: libs/organization/src/lib/components/organization-search/organization-search.component.html [hunk 6]
```diff
@@ -8,16 +8,18 @@
       [formControl]="organizationForm"
       [matAutocomplete]="autoName"
     />
+    <mat-icon color="primary" matSuffix>search</mat-icon>
   </mat-form-field>
   <mat-autocomplete
     #autoName="matAutocomplete"
     [displayWith]="displayFn"
     (optionSelected)="submit($event)"
   >
```
Comment (ioaNikas) [inline libs/organization/src/lib/components/organization-search/organization-search.component.html]: Format

File: styles/variables.scss [hunk 7]
```diff
@@ -41,6 +41,8 @@ $dark-warn: mat-color($blockframes-dark-theme-warn);
   --dynamicIcon: rgb(95, 105, 122);
   --textGrey: rgb(60, 75, 94);
   --toolbarHeight: 66px;
+  --backgroundDisabled: rgb(248,248,249);
+  --textDisabled: rgb(185,194,203);
```
Comment (ioaNikas) [inline styles/variables.scss]: Same as above, we should ask François if we could still use those variables or not.

File: libs/material/src/lib/delivery/components/delivery-stakeholders-repertory/delivery-stakeholders-repertory.component.html (hunk 2 above)
Comment (Hugo-Hontans) [reply libs/material/src/lib/delivery/components/delivery-stakeholders-repertory/delivery-stakeholders-repertory.component.html]: We need to see a difference between the stakeholders who have accepted the request and those who are already waiting.

File: libs/material/src/lib/delivery/components/delivery-stakeholders-repertory/delivery-stakeholders-repertory.component.html (hunk 1 above)
Comment (Hugo-Hontans) [reply libs/material/src/lib/delivery/components/delivery-stakeholders-repertory/delivery-stakeholders-repertory.component.html]: This PR is not yet master, I'm not sure it's a good idea to pre-shot it

File: libs/material/src/lib/delivery/components/delivery-stakeholders-repertory/delivery-stakeholders-repertory.component.scss (hunk 3 above)
Comment (Hugo-Hontans) [reply libs/material/src/lib/delivery/components/delivery-stakeholders-repertory/delivery-stakeholders-repertory.component.scss]: We can do that and fix if we have problems with dark-theme

**Pull request merged**